src/main/webapp/resources/js/pages/admin/components/clients/listing/ClientsTables.jsx

```diff
@@ -56,36 +98,66 @@ export function ClientsTable() {
           </div>
         );
       },
+      width: 250,
     },
     {
-      ...dateColumnFormat(),
-      title: i18n("ClientsTable.column.created"),
-      dataIndex: "createdDate",
+      title: i18n("ClientsTable.column.scope"),
+      dataIndex: ["details", "scope"],
+      render(scopes) {
+        const colors = {
+          read: "cyan",
+          write: "geekblue",
+        };
+        return (
+          <div>
+            {scopes.map((g) => (
```

Comment:
You're going to hate me but change `g` to `scope` :)

Reply:
Ok